CONTRIBUTING.md

```diff
@@ -48,16 +48,16 @@ We welcome icon requests. Before you submit a new issue please make sure the ico
     - For GitHub projects, the amount of "stars" should be above 5k.
     - For anything else, popularity will be judged on a case-by-case basis.
 * Doesn't fall into one of the following categories:
-    - Illegal service (e.g. piracy, malware, threatening material, spam etc.)
+    - Illegal service (e.g. piracy, malware, threatening material, spam, etc.)
```

Comment:
Also, this could be pluralized, right?

```suggestion
    - Illegal services (e.g. piracy, malware, threatening material, spam, etc.)
```

Reply:
Hi, Thanks for pointing out,  small mistake is corrected and the other suggestion is incorporated.

New Commit [bc71aea](https://github.com/simple-icons/simple-icons/pull/7194/commits/bc71aea2b25c1f9ed5945f3bfd178c3dfba582c6)